app/views/shared/_paginator.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="paginator">
+  <ul class="pagination">
+    <li class="<%= previous_status %>">
```

Comment:
This is a really good place to use the `content_tag` syntax. Mixing ruby and erb can be messy and hard to read.

```ruby
<%= content_tag :li, class: previous_state do %>
   ...
<% end %>
```

is just a little nice IMO